packages/terra-form-textarea/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
```

Comment:
We should update this heading to be `## Fixed` since this was incorrect initially